src/components/savedAccounts/savedAccounts.css

```diff
@@ -198,6 +198,18 @@
 
 .unlocked {
   left: 0;
+  padding-top: 25px;
+
+  & span {
+    margin-bottom: 3px;
+  }
+
+  &:hover {
+    & span {
+      transform: scale(1.1);
+      transform-origin: 100%;
```

Comment:
The hover effect would be smoother with `transition: transform ease-in-out 500ms;`